### ISSUE TITLE
22901253: Feature Request: Reminders-like 3D Touch Peek-and-select

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#8353]
#### Description

Phone, Reminders, and Maps app use a different 3D-Touch Peek UI allowing to select an action in one go. For instance, one can force-press on a reminder and select "Remind me on a day" in one go, without releasing the finger.

We'd like to see this exposed as an API.

This is useful for scenarios where the detail view controller focuses on actions that can be applied on the record. Then the user can initiate the action in one go without releasing the finger first. In such a case, showing a detail view controller consisting mainly of action buttons as a preview doesn't really make sense because the preview is non-interactive.

Also, the current model doesn't work well if the detail view controller has a small preferred content height. Then the action buttons are not displayed immediately (even if they'd fit well on the screen) but only after dragging up the preview. The interaction model used in Phone, Reminders, and Maps makes more sense in this case because the actions are more prominently displayed.

So in summary, we'd like to have the ability to:
- show action buttons immediately if they fit above or below the preview
- add an icon to action buttons, similar to the home screen quick actions
- replace "pop" with a second tap gesture on the preview (so to go to the detail view controller, the user has to force-tap, release, and tap, instead of force-tap, and "more-force-tap").
## 

Product Version: 9.1 beta 2
Created: 2015-09-29T19:08:50.751070
Originated: 2015-09-29T21:08:00
Open Radar Link: http://www.openradar.me/22901253
